quality/errorprone/android.gradle

```diff
@@ -1,10 +1,15 @@
 apply plugin: 'net.ltgt.errorprone'
 
 dependencies {
-  annotationProcessor "com.uber.nullaway:nullaway:0.4.6"
-  testAnnotationProcessor "com.uber.nullaway:nullaway:0.4.6"
+  if(JavaVersion.current().java9Compatible) {
```

Comment:
Should be `!JavaVersion.current().java9Compatible)` (the opposite condition).
If the current code works it shows the compiler below is not required :)

Reply:
indeed it should